src/go/rpk/pkg/cli/debug/bundle/bundle.go

```diff
@@ -242,6 +274,20 @@ KUBERNETES
    --logs-since is passed, only the logs within the given timeframe are 
    included.
 
+EXTRA REQUESTS FOR PARTITIONS
```

Comment:
Worth checking with the core team if there are any partitions we should request by default -- or entire topics

Reply:
@bharathv Should we default and always check for any specific topic/partition, like `__consumer_offsets` or something like that?